back.py

```diff
@@ -38,6 +44,24 @@ def play(file):
     soundPlayStop = True
     soundPlaying = False
 
+def call(number):
+    client = Client(account_sid, auth_token)
+    call = client.calls.create(
+                        url='http://demo.twilio.com/docs/voice.xml',
+                        to=number,
+                        from_=source_phone_number
```

Comment:
We need more logs here to monitor the attempt calls

Reply:
sure!